pkg/aspect/logger/logger.go

```diff
@@ -30,10 +32,23 @@ type (
 		Log([]Entry) error
 	}
 
-	// Entry is a set of key-value pairs of attributes that together
-	// constitute the data for log entry. The key is the attribute name,
-	// and the value is a typed value for the named attribute.
-	Entry map[string]interface{}
+	// Entry is the set of data that together constitutes a log entry.
+	Entry struct {
+		// LogName is the name of the log in which to record this entry.
+		LogName string
+		// Labels are a set of metadata associated with this entry.
+		// For instance, Labels can be used to describe the monitored
```

Comment:
Let's start with just Labels for now (which supports monitored resources indirectly as is). We can add monitored resources in a subsequent CL, when we have a clearer understanding of whether they should exist across all adapters or not.